arches/management/commands/es.py

```diff
@@ -125,7 +125,8 @@ def add_arguments(self, parser):
             action="store",
             dest="clear_index",
             default=True,
-            help="Set to True(default) to remove all the resources from the index before the reindexing operation",
+            type=lambda x: str(x).lower() == "true",
```

Comment:
We have a helper `str_to_bool()` that's a little more permissive (accepts "t", "y", and "yes"), but this works too and I'm all for just changing this to be more sane in the next version.

Reply:
done